### ISSUE TITLE
Emulator : Remove the bar from the screenshot

### ISSUE THREAD

**Issue by Progi1984** [mozilla/r2d2b2g#440]
Following this bug #421 , i create this ticket.

Why ? A screenshot in the emulator seems like in a real phone, so no bar. And it's more easier to make a screenshot for putting on the Marketplace.

Poke @Gioyik

**Comment by Gioyik** [comment]
I understand now the problem. I reproduce it. Well yes, take a "native screenshot" with the simulator shows you the bottom bar with the home icon. Not good :(

I think that this could be marked as "feature-request".

Ping @mykmelez

**Comment by Progi1984** [comment]
Thanks :)

**Comment by mykmelez** [comment]
Indeed, a screenshot taken within the simulation shouldn't show the toolbar at the bottom of the Simulator window!

**Comment by Gioyik** [comment]
So, @mykmelez Could be possible to think this as a feature in the future for the simulator? Take a creenshot without the bottom bar? :)

**Comment by mykmelez** [comment]
@Gioyik I think of it more as a bug than a feature: it isn't intended for the bottom bar to be in these screenshots, so it shouldn't be, which makes its presence a bug!

**Comment by Gioyik** [comment]
Thanks @mykmelez
